site.yml

```diff
@@ -23,6 +23,15 @@
   tags:
     - always
 
+- hosts: consul
```

Comment:
Instead of using ``consul`` as a group we could create an alias in our inventory called ``sensu``.